walk-in-interview/src/main/java/com/google/configuration/ProductionFireStoreConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package com.google.configuration;
+
+/** Class that specify the configuration details for firestore during production phase. */
+public final class ProductionFireStoreConfiguration implements FireStoreConfiguration {
+    private static ProductionFireStoreConfiguration developmentFireStoreConfiguration = null;
+
+    private static final String PROJECT_ID = "";
+
+    private ProductionFireStoreConfiguration() {}
+
+    public static ProductionFireStoreConfiguration getFireStoreConfiguration() {
+        if (developmentFireStoreConfiguration == null) {
```

Comment:
Optionally, if it's not implemented at this stage, you can throw not implemented exception.